### ISSUE TITLE
Request Is there any kinda tool I could use

### ISSUE THREAD

**Issue by Flukethoughts** [AHAAAAAAA/PokemonGo-Map#3188]
Hello, This is more of a request is there any kinda tool I could use to generate a lot fo GPS cords that would overlap and cover a large area. Basic I would drag a box over the map, and it would generate the cords needed to cover the box I created. I would like to use more then 500 accounts and just have to use the GPS cords given by the program described. Please if anyone can help that would be wonderful.

**Comment by weigel23** [comment]
Have a look at the Hex-Beehive-Generator 

https://github.com/AHAAAAAAA/PokemonGo-Map/tree/master/Tools/Hex-Beehive-Generator

**Comment by Caspiel86** [comment]
i have been looking for a preview tool for beehive. by any chance do you know of one. i want to make sure the size fits the city i am using with out making some of the workers search the ocean.

**Comment by Flukethoughts** [comment]
@weigel23 after using the beehive could u go into more details about the steps it uses what is the default steps. thank you

**Comment by brettgus** [comment]
I agree, it would be nice to have a tool where you could draw a polygon and it would discard workers lat/lon outside those points.
